hack/tools/pkg/janitor/janitor.go

```diff
@@ -197,6 +207,118 @@ func (s *Janitor) deleteVSphereVMs(ctx context.Context, folder string) error {
 	return nil
 }
 
+// DeleteCNSVolumes deletes all volumes from tests.
+func (s *Janitor) DeleteCNSVolumes(ctx context.Context, boskosResourceName string) error {
+	log := ctrl.LoggerFrom(ctx).WithName("volumes")
```

Comment:
Usually we don't really use named loggers like this (this will add a prefix to the message)

Reply:
Usually not but in the janitor we do

Reply:
Strange :D